core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -888,6 +888,18 @@
 
   @BaseMessage("Not a valid input for JSON_STORAGE_SIZE: ''{0}''")
   ExInst<CalciteException> invalidInputForJsonStorageSize(String value);
+
+  @BaseMessage("Table function is not allowed in select list in current SQL conformance level")
+  ExInst<SqlValidatorException> notAllowTableFunctionInSelect();
+
+  @BaseMessage("''{0}'' should be a table function")
+  ExInst<SqlValidatorException> exceptTableFunction(String name);
+
```

Comment:
`exceptTableFunction` -> `notTableFunctionError` ?

Reply:
Well this exception means we  except a table function for `func` in `select func() as (t0)`.